.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   qa:
     working_directory: /go/src/github.com/grafana/metrictank
     docker:
-      - image: circleci/golang:1.11.4
+      - image: circleci/golang:1.12
```

Comment:
now we're going to build with a different image than we test with. as long as this is only true for those test utilities i guess that's fine, as long as we don't also change the unit tests to this image and leave the build at the older one.

Reply:
Yeah, since it was only the QA tests I feel comfortable with this.